### PR TITLE
Display node allocated resources in graphs instead of table

### LINKED ISSUE #1997: Show allocated resources graphs on node details page
Show graphs instead of:

zrzut ekranu 2017-05-19 o 12 36 12

### PR DESCRIPTION
**Pull request by maciaszczykm**
Current state:

zrzut ekranu 2017-06-05 o 15 47 59

- [x] fix node list view console error
- [x] position all three graphs well
- [x] third graph should have only one ring
- [x] add legends
- [x] change table to something smaller

Closes #1997.

### CODE REVIEW

**Comment by maciaszczykm** [comment]
Current state:

zrzut ekranu 2017-06-02 o 13 24 04

**Comment by cheld** [comment]
Very nice!

but...just looking at the picture..what are the graphs showing? what is green, what is blue? why three circles?
Maybe it is obvious when using it, but just looking at the picture, I could not guess it...

**Comment by maciaszczykm** [comment]
@cheld I know, now I will focus on adding legend to make it easier to understand. Unfortunately auto-generated legend doesn't work for us as there are two graphs merged into one. I will figure something out. I will add title too.

 I am glad, that you like current look.

File: src/app/frontend/node/detail/allocatedresources.html [hunk 1]
```diff
@@ -14,69 +14,73 @@
...
-      </kd-resource-card-column>
+<kd-content-card>
+  <kd-title>[[Allocated resources|Label 'Allocated resources' for the allocated resources section.]]</kd-title>
+  <kd-content>
+    <div style="display: flex; flex-flow: row wrap; justify-content: center">
+      <div style="flex: 1 auto; text-align: center">
+        <kd-node-graph requests="$ctrl.allocatedResources.cpuRequestsFraction"
+                       limits="$ctrl.allocatedResources.cpuLimitsFraction"></kd-node-graph>
+      </div>
+      <div style="flex: 1 auto; text-align: center">
```
Comment (bryk) [inline src/app/frontend/node/detail/allocatedresources.html]: Hmm... Inline styles? Can we put them in CSS?

File: src/app/frontend/node/detail/allocatedresources.html [hunk 2]
```diff
@@ -14,69 +14,73 @@
...
+        <kd-node-graph requests="$ctrl.allocatedResources.memoryRequestsFraction"
+                       limits="$ctrl.allocatedResources.memoryLimitsFraction"></kd-node-graph>
+        <!-- TODO(maciaszczykm): Translude data to show legend. -->
+      </div>
+      <div style="flex: 1 auto; text-align: center">
```
Comment (bryk) [inline src/app/frontend/node/detail/allocatedresources.html]: Hmm... Inline styles? Can we put them in CSS?

Comment (maciaszczykm) [inline src/app/frontend/node/detail/allocatedresources.html]: Yes, still work in progress. I will do it :)

**Comment by maciaszczykm** [comment]
PTAL

**Comment by maciaszczykm** [comment]
@floreks Sounds like a plan to me 👍

**Pull request merged**